vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -95,12 +108,19 @@ def build(self):
             fluid=True,
         )
 
-    def _make_page_layout(self, page: Page):
+    def _get_page_divs(self, page: Page) -> PageDivs:
```

Comment:
Splitting this function out should enable the user to create any kind of layout more easily by overwriting the `_make_page_layout` function and re-arranging the elements from the TypedDict returned by `self._get_page_divs`. Some custom CSS might be required as well, but at least the re-arrangement should be easier now.